cmd/importer/pod/check.go

```diff
@@ -32,40 +32,45 @@ import (
 func Check(ctx context.Context, c client.Client, cache *util.ImportCache, jobs uint) error {
 	ch := make(chan corev1.Pod)
 	go func() {
-		err := util.PushPods(ctx, c, cache.Namespaces, cache.QueueLabel, ch)
+		err := util.PushPods(ctx, c, cache.Namespaces, ch)
 		if err != nil {
 			ctrl.LoggerFrom(ctx).Error(err, "Listing pods")
 		}
 	}()
-	summary := util.ConcurrentProcessPod(ch, jobs, func(p *corev1.Pod) error {
+	summary := util.ConcurrentProcessPod(ch, jobs, func(p *corev1.Pod) (bool, error) {
 		log := ctrl.LoggerFrom(ctx).WithValues("pod", klog.KObj(p))
 		log.V(3).Info("Checking")
 
-		cq, err := cache.ClusterQueue(p)
-		if err != nil {
-			return err
+		cq, skip, err := cache.ClusterQueue(p)
+		if skip || err != nil {
+			return skip, err
 		}
 
 		if len(cq.Spec.ResourceGroups) == 0 {
-			return fmt.Errorf("%q has no resource groups: %w", cq.Name, util.ErrCQInvalid)
+			return false, fmt.Errorf("%q has no resource groups: %w", cq.Name, util.ErrCQInvalid)
 		}
 
 		if len(cq.Spec.ResourceGroups[0].Flavors) == 0 {
-			return fmt.Errorf("%q has no resource groups flavors: %w", cq.Name, util.ErrCQInvalid)
+			return false, fmt.Errorf("%q has no resource groups flavors: %w", cq.Name, util.ErrCQInvalid)
 		}
 
 		rfName := string(cq.Spec.ResourceGroups[0].Flavors[0].Name)
 		rf, rfFound := cache.ResourceFalvors[rfName]
 		if !rfFound {
-			return fmt.Errorf("%q flavor %q: %w", cq.Name, rfName, util.ErrCQInvalid)
+			return false, fmt.Errorf("%q flavor %q: %w", cq.Name, rfName, util.ErrCQInvalid)
+		}
+
+		var pv int32
+		if pc, found := cache.PriorityClasses[p.Spec.PriorityClassName]; found {
+			pv = pc.Value
 		}
```

Comment:
error if the `PriorityClassName!=""` and it is not found

Reply:
done